src/mapeo-manager.js

```diff
@@ -654,13 +665,15 @@ export class MapeoManager extends TypedEmitter {
     return this.#invite
   }
 
+  // TODO: Can we remove this?
   /**
    * @param {import('./media-server.js').StartOpts} [opts]
    */
   async startMediaServer(opts) {
     await this.#mediaServer.start(opts)
   }
 
+  // TODO: Can we remove this?
   async stopMediaServer() {
     await this.#mediaServer.stop()
   }
```

Comment:
highlighting these - can these now be removed or is it still helpful to have them?

Reply:
no, can be removed. Needs change to CoMapeo code.